packages/server/tests/ListUserFollowers.test.ts

```diff
@@ -197,7 +197,7 @@ test.group('List user followers tests group', (group) => {
                 fill: (index) => ({
                     uuid: datatype.uuid(),
                     nickname: `${index % 2 === 0 ? searchQuery : ''}${unique(
-                        () => internet.userName(),
+                        () => internet.userName().toLowerCase(),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** 🙈